cetloxblock/cetloxblock/cetloxblock.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+""" 
+This XBlock displays a CET LO item.
+The studio view enables the author to select the LO for the current xblock instance.
+"""
+
+import pkg_resources
+#import urllib
+from xblock.core import XBlock
+#from xblock.fields import String, Scope, ScopeBase, UserScope, BlockScope

```

Comment:
Similar comment as above -- we should keep commented-out code out of the repository, until it's actually needed.